main.go

```diff
@@ -215,80 +214,80 @@ func prepareDir(dir string) error {
 	return nil
 }
 
-func getModulePackagePath(goPath, outputPath string) string {
-	pwd, err := os.Getwd()
-	if err != nil {
-		log.Fatalf("Error getting current directory: %s", err.Error())
+// parseCmdConfig determines the which code is generated and the location of the generated code
+// from the command line arguments
+func (c *config) parse() error {
+	if err := c.setGoPaths(swag.StringValue(c.outputPath), swag.StringValue(c.goPackageName)); err != nil {
+		return err
 	}
-	goSrcPath := fmt.Sprintf("%v%v", goPath, "/src/")
-	return path.Join(strings.TrimPrefix(pwd, goSrcPath), outputPath)
-}
 
-// getModulePackageName gets the package name of the generated code
-// Example: if packagePath = github.com/Clever/wag/v6/gen-go and the module name is github.com/Clever/wag/v6/v2
-// the function will return github.com/Clever/wag/v6/v2/gen-go
-// Example: if packagePath = github.com/Clever/wag/v6/gen-go and the module name is github.com/Clever/wag/v6
-// the function will return  github.com/Clever/wag/v6/gen-go
-func getModulePackageName(modFile *os.File, outputPath string) string {
-	// read first line of module file
-	r := bufio.NewReader(modFile)
-	b, _, err := r.ReadLine()
-	if err != nil {
-		log.Fatalf("Error checking module name: %s", err.Error())
+	clientLanguage, jsModulePath := swag.StringValue(c.clientLanguage), swag.StringValue(c.jsModulePath)
+	if swag.BoolValue(c.clientOnly) && swag.BoolValue(c.dynamoOnly) {
+		return fmt.Errorf("Cannot use -dynamo-only and -client-only together")
+	} else if swag.BoolValue(c.clientOnly) {
+		c.generateServer = false
```

Comment:
Do you think it's important to have this extra verbosity, given that `bool` fields all default to `false` anyway?

Reply:
good point. I don't think it's necessary anymore. It was an artifact of the pre-refactor code which set all of this to true.